.github/workflows/ci.yml

```diff
@@ -130,6 +130,43 @@ jobs:
           GO_ARCH: linux-amd64
         run: ./scripts/ci-docker-run
 
+  ubuntu-2404:
+    name: debbuild-ubuntu24
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v2
+      # fetch tags as checkout@v2 doesn't do that by default
+      - run: git fetch --prune --unshallow --tags --force
+
+      - name: Build and test deb under docker
+        env:
+          OS_TYPE: ubuntu
+          OS_VERSION: 24.04
+          GO_ARCH: linux-amd64
+        run: ./scripts/ci-docker-run
+
+  rpmbuild-centos7:
+    runs-on: ubuntu-22.04
+    name: rpmbuild-centos7
+    steps:
+      - uses: actions/checkout@v2
+      # fetch tags as checkout@v2 doesn't do that by default
+      - run: git fetch --prune --unshallow --tags --force
+
+      - name: Build and test rpm under docker
+        env:
+          OS_TYPE: centos
+          OS_VERSION: 7
+          GO_ARCH: linux-amd64
+        run: ./scripts/ci-docker-run
+
+      - name: Install and test unprivileged
+        env:
+          OS_TYPE: centos
+          OS_VERSION: 7
+          TEST_TYPE: unpriv
+        run: ./scripts/ci-docker-run
+
```

Comment:
Please remove the centos7 build.  We're not supporting that anymore as of this morning.

Reply:
done, thanks!